src/pudl/extract/ferc1.py

```diff
@@ -723,11 +728,28 @@ def extract_dbf(
             f"Converting extracted FERC Form 1 table {pudl_table} into a "
             f"pandas DataFrame from DBF table."
         )
-        ferc1_raw_dfs[pudl_table] = extract_dbf_generic(
-            ferc1_engine=sa.create_engine(pudl_settings["ferc1_db"]),
-            ferc1_settings=ferc1_settings,
-            table_name=TABLE_NAME_MAP[pudl_table]["dbf"],
-        )
+        if pudl_table == "income_statement_ferc1":
+            # special case for the income statement. bc the dbf table is two tables.
+            ferc1_raw_dfs[pudl_table] = pd.concat(
+                [
+                    extract_dbf_generic(
+                        ferc1_engine=sa.create_engine(pudl_settings["ferc1_db"]),
+                        ferc1_settings=ferc1_settings,
+                        table_name=TABLE_NAME_MAP[pudl_table]["dbf"][0],
+                    ),
+                    extract_dbf_generic(
+                        ferc1_engine=sa.create_engine(pudl_settings["ferc1_db"]),
+                        ferc1_settings=ferc1_settings,
+                        table_name=TABLE_NAME_MAP[pudl_table]["dbf"][1],
+                    ),
+                ]
+            )
+        else:
+            ferc1_raw_dfs[pudl_table] = extract_dbf_generic(
+                ferc1_engine=sa.create_engine(pudl_settings["ferc1_db"]),
+                ferc1_settings=ferc1_settings,
+                table_name=TABLE_NAME_MAP[pudl_table]["dbf"],
+            )
```

Comment:
@bendnorman ... this might change in review but here is my rough draft of how i'm dealing with two dbf tables feeding into one pudl table. I decided to do *one* special case and just do it here in the extract step instead of integrating this special case into the extract step (like having `ferc1_raw_dfs[pudl_table]` have two tables) and have the transform step do a special case concatenation.

Reply:
This seems fine for now! I think we can have a node in the dag that extracts all of the raw dbf tables then have a downstream node merge `f1_income_stmnt` and `f1_incm_stmnt_2` to create `income_statement_ferc1`.

Reply:
awesome! yes that sounds nicer and this will be easy to rip out when we get there